src/libtest/lib.rs

```diff
@@ -22,7 +22,7 @@
 #![unstable(feature = "test", issue = "27812")]
 #![doc(html_root_url = "https://doc.rust-lang.org/nightly/", test(attr(deny(warnings))))]
 #![feature(asm)]
-#![cfg_attr(any(unix, target_os = "cloudabi"), feature(libc, rustc_private))]
+#![feature(libc, rustc_private)]
```

Comment:
nit: If we're dropping the condition here I think we should continue to list them one-by-one, so:
```suggestion
#![feature(libc)]
#![feature(rustc_private)]
```